root/static/scripts/common/constants/locales.json

```diff
@@ -52,6 +52,7 @@
   "bas_CM":"Basaa Cameroon",
   "be":"Belarusian",
   "be_BY":"Belarusian Belarus",
+  "be_tarask":"Belarusian",
```

Comment:
It's a bit weird for `be_tarask` to also be just "Belarusian". It's the old no-longer-official orthography (https://en.wikipedia.org/wiki/Belarusian_orthography_reform_of_1933) and it should probably be marked in some way. Can we hand-edit this to name it "Belarusian Taraškievica" or something?

Reply:
Hand-editing `locales.json` would just make Node code to return different name and would not fix the name returned by the Perl code which uses `DateTime::Locale` directly. I rather reported the issue to `DateTime::Locale` (might be an upstream issue of CLDR though).

Reply:
To be fixed with the pull request https://github.com/metabrainz/musicbrainz-server/pull/2606.